catalyst_voices/lib/pages/widgets/voices_app_bar/actions/notifications_indicator.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:flutter/material.dart';
+
+/// A [NotificationsIndicator] widget that is used to display pending 
+/// notifications.
+class NotificationsIndicator extends StatelessWidget {
+  final String? badgeText;
+  final VoidCallback? onPressed;
+
+  const NotificationsIndicator({
+    super.key,
+    this.badgeText,
+    this.onPressed,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return badgeText != null
+        ? Badge(
+            label: Text(badgeText!),
+            child: _iconButton,
+          )
+        : _iconButton;
+  }
+
+  Widget get _iconButton => IconButton.outlined(
```

Comment:
Don't want to be picky but refactor to widget, please

Reply:
Alright, will do in the next PR 👍